src/main/java/com/minecolonies/coremod/client/gui/blockui/RotatingItemIcon.java

```diff
@@ -0,0 +1,110 @@
+package com.minecolonies.coremod.client.gui.blockui;
+
+import com.ldtteam.blockui.controls.ItemIcon;
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.world.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Variation of the ItemIcon which accepts multiple items and slowly rotates between them.
+ */
+public class RotatingItemIcon extends ItemIcon
```

Comment:
Feel like this should be in BlockUI

Reply:
Intentionally here, on my to-do list

Reply:
Okay, then just add a "todo to be moved to blockui by nightenom" here